.nengobones.yml

```diff
@@ -0,0 +1,59 @@
+pkg_name: nengo
+repo_name: nengo/nengo
+
+ci_scripts:
+  - template: static
+    pip_install:
+      - flake8
+    post_commands:
+      - flake8 nengo
+      - flake8 --ignore=E703,W391 docs
+  - template: test
+    conda_install:
+      - $NUMPY_VERSION
+  - template: coverage
+    output_name: test-coverage
+    conda_install:
+      - $NUMPY_VERSION
+    coverage: true
+  - template: coverage
+    output_name: test-coverage-scipy
+    conda_install:
+      - $NUMPY_VERSION
+      - scipy
+    pip_install:
+      - scikit-learn
+    coverage: true
+  - template: docs
+    conda_install:
+      - $NUMPY_VERSION
+      - scipy
+      - matplotlib
+
+travis_yml:
```

Comment:
Does order matter in this file? I ask because it would be nice to move this before `ci_scripts` because it defines `NUMPY_VERSION`, which is used in `ci_scripts`.

Reply:
In the interest of providing a rapid response I will say: I think the order doesn't matter.